### DISCUSSION TITLE
ShipStation Integration Not Working

### DISCUSSION THREAD

**Discussion by sbpowell** [swellstores/community#553] [Help] [answered]
My client has a store that's connected to a variety of other inventory and shipment services. When an order is created in Swell it's passed via webhook to another service, eventually landing at ShipStation. Our hope is to make sure that when an order is shipped in ShipStation it'll use the webhook integration to send the shipping price and tracking number back to Swell. However, it appears this just isn't happening. 

What does the ShipStation integration use to connect order IDs? My best guess at this point is ShipStation orders have different order IDs in a format derives from another service that isn't common with Swell. Any help is appreciated!

**Comment by logeshswell** [comment]
Can you reach out to support@swell.is with your store-id and the screen recording of the issue that you experience?

**Reply by logeshswell** [reply, marked as answer]
`tracking_code` is used to map Swell data with Shipstation.

**Comment by sbpowell** [comment]
Hi there. The store id is recovercann.

If tracking_code is used to map Swell data, does that mean the Shipstation
integration can't create a new shipment in Swell? Our flow is that once an
order is shipped in ShipStation it should create a shipment with the tracking
number and shipping price in Swell, ideally.

Spencer

**Reply by logeshswell** [reply]
Basically, if the Swell order is paid successfully, then Swell would trigger the `order.paid` webhook to Shipstation and create an order in Shipstation as well.
Once you create a label and fulfill a shipment for an order in Shipstation, Shipstation will trigger `SHIP_NOTIFY` webhook to Swell and Swell will mark the order as Fulfilled with shipment details like tracking_code, carrier_name, service_name, etc.

**Reply by logeshswell** [reply]
So, there should be an order in the Swell system before it is fulfilled in Shipstation.
